src/modules/visits/types.ts

```diff
@@ -53,6 +56,7 @@ export interface GqlUpdateVisit {
 	end_date: string;
 	status: VisitStatus;
 	updated_by?: string;
+	access_type?: string;
 }
```

Comment:
always prefer the direct database type since it will avoid property naming mistakes like:
updated_by vs last_updated_by

```
export type GqlUpdateVisit = UpdateVisitMutation['update_maintainer_visitor_space_request_by_pk'];
```